pkg/client/unversioned/util.go

```diff
@@ -77,3 +79,13 @@ func RetryOnConflict(backoff wait.Backoff, fn func() error) error {
 	}
 	return err
 }
+
+func isEnabledVersion(group string, gv *unversioned.GroupVersion) bool {
```

Comment:
I must have missed it there, but I do remember looking for it. Will update. Thx!